lib/core/src/lib/auth/basic-auth/basic-alfresco-auth.service.ts

```diff
@@ -317,14 +317,27 @@ export class BasicAlfrescoAuthService extends BaseAuthenticationService {
         return this.redirectUrl && (this.redirectUrl.provider === 'ALL' || provider === 'ALL');
     }
 
+    /**
+     * @deprecated use `getUsername()` instead
```

Comment:
@MichalKinas what do you think - should we remove all these functions now or should we leave them as deprecated for now?

Reply:
They were just marked as deprecated, I would leave them now and remove in next one or two RTs